quic/s2n-quic-core/src/connection/limits.rs

```diff
@@ -73,8 +74,12 @@ impl Default for Limits {
 }
 
 macro_rules! setter {
-    ($name:ident, $field:ident, $inner:ty) => {
+    ($name:ident, $field:ident, $inner:ty $(, $validate_value:ident, $validaiton:tt)?) => {
```

Comment:
Let's make this look more like a closure so the invocation makes more sense

```suggestion
    ($name:ident, $field:ident, $inner:ty $(, |$validate_value:ident| $validaiton:block)?) => {
```